### PR TITLE
[W4.11e][W16-1]Travis Ching Jia Yea

### PR DESCRIPTION
**Pull request by GilgameshTC**
automate testing on Utils.isAnyNull method

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -34,6 +34,23 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+    }
+
+    @Test
+    public void isAnyNull() throws Exception {
```
Comment (bqnguyen94) [inline test/java/seedu/addressbook/common/UtilsTest.java:41]: I think you shouldn't name the method `isAnyNull` since there already exists a method with the same name in [here](https://github.com/se-edu/addressbook-level2/blob/master/src/seedu/addressbook/common/Utils.java#L15).

Comment (bqnguyen94) [reply test/java/seedu/addressbook/common/UtilsTest.java:41]: Also, one more test case you can try is empty `Objects` parameter.

**Review by bqnguyen94 — COMMENTED** [review]
Looks good 👍 please close the PR after reading.
